featured-projects/README.md

```diff
@@ -0,0 +1,17 @@
+# Featured projects
+
+## Lotus
+Lotus is an implementation of the Filecoin Distributed Storage Network specification.
+You can find the Lotus Testground plans [here](https://github.com/filecoin-project/lotus/).
+
+## libp2p
+libp2p is an open source networking stack and library modularized out of The IPFS Project, and bundled separately for other tools to use.
+You can find the libp2p Testground plans [here](https://github.com/libp2p).
```

Comment:
The link leads to the organization.

Also, `You find the plans in the [Testground Plans Repository]().` would be better for accessibility. Check out the [`What’s wrong with using "Click Here" links?` article](https://webaccess.berkeley.edu/ask-pecan/click-here).